geoportailv3/static/js/olcs/lux3dmanager.js

```diff
@@ -24,10 +24,27 @@ app.olcs.Lux3DManager = class extends ngeo.olcs.Manager {
      */
     this.ngeoLocation_ = ngeoLocation;
 
+    /*
+     * A factor used to increase the screen space error of terrain tiles when they are partially in fog. The effect is to reduce
+     * the number of terrain tiles requested for rendering. If set to zero, the feature will be disabled. If the value is increased
+     * for mountainous regions, less tiles will need to be requested, but the terrain meshes near the horizon may be a noticeably
+     * lower resolution. If the value is increased in a relatively flat area, there will be little noticeable change on the horizon.
+     * type {Number}
+     * default 2.0
+     */
     if (ngeoLocation.hasParam('fog_sse_factor')) {
       this.fogSSEFactor = parseFloat(ngeoLocation.getParam('fog_sse_factor'));
     }
 
+    /*
+     * A scalar that determines the density of the fog. Terrain that is in full fog are culled.
+     * The density of the fog increases as this number approaches 1.0 and becomes less dense as it approaches zero.
+     * The more dense the fog is, the more aggressively the terrain is culled. For example, if the camera is a height of
+     * 1000.0m above the ellipsoid, increasing the value to 3.0e-3 will cause many tiles close to the viewer be culled.
+     * Decreasing the value will push the fog further from the viewer, but decrease performance as more of the terrain is rendered.
+     * type {Number}
+     * default 2.0e-4
```

Comment:
again, not [this one](https://github.com/openlayers/ol-cesium/blob/master/src/olcs/contrib/manager.js#L75) ?